packages/stopwatch/tsconfig.json

```
@@ -0,0 +1,10 @@
+{
+	"extends": "../ts-config/src/tsconfig.json",
+	"compilerOptions": {
+		"target": "ES2020",
+		"module": "ESNext",
+		"outDir": "build",
+		"rootDir": "src"
```

```suggestion
		"outDir": "build",
		"rootDir": "src",
		"composite": true,
		"tsBuildInfoFile": "./dist/.tsbuildinfo"
```

This has to build as CommonJS